### PR TITLE
fix: sb component input handling

### PR DESCRIPTION
**Pull request by dipankarmaikap**
This PR improves `<sb-component>` input handling by making it more flexible and resilient.

It now supports `SbBlokData`, `SbBlokData[]`, `null`, and `undefined`, and normalizes all values internally into a consistent array format.

## Changes

* Support single and multiple blok inputs
* Handle `null` and `undefined` safely
* Normalize input internally before rendering
* Reduce need for manual guards in templates

## Why

Improves DX and makes the component safer and easier to use.

### CODE REVIEW

File: packages/angular/src/lib/blok/sb-component.component.ts [hunk 1]
```diff
@@ -1,17 +1,27 @@
...
-    @for (blok of sbBloks(); track blok._uid) {
+    @for (blok of bloks(); track blok._uid) {
       <ng-container [sbBlok]="blok" />
     }
   `,
-  imports: [SbBlokDirective],
 })
 export class StoryblokComponent {
-  readonly sbBloks = input.required<SbBlokData[]>();
+  readonly sbBlok = input<SbBlokData | SbBlokData[] | null | undefined>();
```
Comment (alexjoverm) [inline packages/angular/src/lib/blok/sb-component.component.ts:16]: @dipankarmaikap shall we can add a test? Something simple, just input/output based to ensure it works with all formats (object, array, null, etc)

**Review by alexjoverm — APPROVED** [review]
LGTM with one comment

File: packages/angular/src/lib/blok/sb-component.component.ts (hunk 1 above)
Comment (alexjoverm) [reply packages/angular/src/lib/blok/sb-component.component.ts:16]: and btw, questioning here if we want to name it `block` (see team internal message)
